app/connectors/connectors/mqtt/src/main/resources/META-INF/syndesis/connector/paho.json

```diff
@@ -0,0 +1,110 @@
+{
+    "id": "paho",
+    "name": "MQTT",
+    "description": "Publish and Subscribe Messages to a MQTT broker.",
+    "icon": "paho",
+    "dependencies": [
+        {
+            "type": "MAVEN",
+            "id": "@project.groupId@:@project.artifactId@:@project.version@"
+        }
+    ],
+    "componentScheme": "paho",
+    "configuredProperties": {
+        
+    },
+    "tags": [
+        "verifier"
+    ],
+    "properties": {
+        "brokerUrl": {
+            "kind": "parameter",
+            "displayName": "MQTT broker URL",
+            "group": "common",
+            "required": true,
+            "type": "string",
+            "javaType": "java.lang.String",
+            "deprecated": false,
+            "secret": false,
+            "description": "MQTT broker URL (eg 'tcp://localhost:1833')"
+        }
+    },
+    "actions": [
+        {
+            "name": "Publish",
+            "description": "Publish a message to a MQTT Broker",
+            "id": "io.syndesis:paho-publish-connector",
+            "pattern": "To",
+            "actionType": "connector",
+            "descriptor": {
+                "componentScheme": "paho",
```

Comment:
if `componentScheme` is set as connector level, it is not needed to define it again

Reply:
Done, please review again. Thanks, Luca.